backend/src/test/java/de/silvia/backend/security/services/UserServiceTest.java

```diff
@@ -18,7 +20,9 @@
     @Test
     public void loadUserByUsername()  {
```

Comment:
it helps to include your expectation for your test in the name, e.g. shouldReturnUserForGivenUsername, otherwise the intent of the test is unclear. Also it helps to differentiate test scenarios when e.g. you introduce tests where you expect an error

Reply:
Good hint, thanks!